onboard/catkin_ws/src/cv/models/models.yaml

```diff
@@ -0,0 +1,8 @@
+buoy:
```

Comment:
Should there be a model for the gate?

Reply:
Yes -- we have not added the model yet. We'll train and add it a bit later; right now we're still finishing up the basic functionality for this pull request